app/controllers/order_imports_controller.rb

```diff
@@ -15,23 +15,23 @@ def initialize
 
   def new
     @order_import = OrderImport.new
+    @order_imports = get_order_imports.paginate(page: params[:page])
   end
 
   def create
     begin
       process_order_import!
-      flash_import_result
     rescue => e
       import_exception_alert(e)
     end
 
-    render action: 'show'
+    redirect_to new_facility_order_import_path
   end
 
   private
 
-  def flash_now(key, message)
-    flash.now[key] = truncate(message, length: 2048)
+  def set_flash(key, message)
+    flash[key] = truncate(message, length: 2048)
```

Comment:
Do we need to worry about this truncation anymore? If not, it might be nice to just go back to plain `flash[:error] = I18n.t(...)` instead of this extracted method.